packages/rum-core/src/boot/startRum.ts

```diff
@@ -96,7 +97,14 @@ export function startRumEventCollection(
   const parentContexts = startParentContexts(lifeCycle, session)
   const urlContexts = startUrlContexts(lifeCycle, locationChangeObservable, location)
   const foregroundContexts = startForegroundContexts()
-  const batch = startRumBatch(configuration, lifeCycle)
+
+  let stopBatch: () => void
+
+  if (isEventBridgePresent()) {
+    startRumEventBridge(lifeCycle)
```

Comment:
NIT: maybe expose a stop function to unsubscribe from the lifecycle

Reply:
It could simplify the code, but it's a bit weird to me since we never unsubscribe from lifecycle. Should we start doing it to make RUM stoppable?

Reply:
This should be discussed further, so you may ignore my comment for this PR, no problem :)

Reply:
okay :)